sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -232,6 +242,99 @@ export class AssessmentController {
     }
   }
 
+  /**
+   * Verify as per the order of original assessment study start date if this assessment
+   * is the first in the sequence to be calculated
+   * for the given student inside the given program year.
+   * If an assessment is identified to be the first in sequence to be calculated
+   * then set calculation start date to hold other assessments until the calculation
+   * is complete.
+   */
+  @ZeebeWorker(Workers.VerifyAssessmentCalculationOrder, {
+    fetchVariable: [ASSESSMENT_ID],
+    maxJobsToActivate: MaxJobsToActivate.Normal,
+  })
+  async verifyAssessmentCalculationOrder(
+    job: Readonly<
+      ZeebeJob<
+        AssessmentDataJobInDTO,
+        ICustomHeaders,
+        VerifyAssessmentCalculationOrderJobOutDTO
+      >
+    >,
+  ): Promise<MustReturnJobActionAcknowledgement> {
+    const jobLogger = new Logger(job.type);
+    try {
+      const assessment = await this.studentAssessmentService.getById(
+        job.variables.assessmentId,
+      );
+      if (!assessment) {
+        const message = "Assessment not found.";
+        jobLogger.error(message);
+        return job.error(ASSESSMENT_NOT_FOUND, message);
+      }
+      const assessmentId = job.variables.assessmentId;
+      const studentId = assessment.application.student.id;
+      const programYearId = assessment.application.programYear.id;
+      jobLogger.log(
+        `Verifying the assessment calculation order for assessment: ${assessmentId} student:${studentId} program year: ${programYearId}.`,
```

Comment:
Can we change the below log as:
"Verifying the assessment calculation order for assessment id ${assessmentId}, student id${studentId}, and program year is ${programYearId}."